[2024-11-26T07:20:33.644+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-26T07:20:33.665+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_weather.extract_data scheduled__2024-11-25T00:00:00+00:00 [queued]>
[2024-11-26T07:20:33.674+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_weather.extract_data scheduled__2024-11-25T00:00:00+00:00 [queued]>
[2024-11-26T07:20:33.676+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-26T07:20:33.690+0000] {taskinstance.py:2888} INFO - Executing <Task(WeatherToCSVOperator): extract_data> on 2024-11-25 00:00:00+00:00
[2024-11-26T07:20:33.698+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=429) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-26T07:20:33.698+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_weather', 'extract_data', 'scheduled__2024-11-25T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/extract_weather.py', '--cfg-path', '/tmp/tmp5enpyuvr']
[2024-11-26T07:20:33.701+0000] {standard_task_runner.py:105} INFO - Job 73: Subtask extract_data
[2024-11-26T07:20:33.701+0000] {standard_task_runner.py:72} INFO - Started process 436 to run task
[2024-11-26T07:20:33.717+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-26T07:20:33.755+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_weather.extract_data scheduled__2024-11-25T00:00:00+00:00 [running]> on host ac9b5e0c2e2c
[2024-11-26T07:20:33.757+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-26T07:20:33.853+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_weather' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-25T00:00:00+00:00'
[2024-11-26T07:20:33.856+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-26T07:20:33.881+0000] {operator.py:23} INFO - Fetching weather in Moscow
[2024-11-26T07:20:33.897+0000] {base.py:84} INFO - Retrieving connection 'weather_api'
[2024-11-26T07:20:34.358+0000] {operator.py:26} INFO - Create DataFrame
[2024-11-26T07:20:34.360+0000] {operator.py:29} INFO - Writing weather to /opt/***/data/Moscow_20241126.csv
[2024-11-26T07:20:34.367+0000] {operator.py:32} INFO - Weather data written to /opt/***/data/Moscow_20241126.csv
[2024-11-26T07:20:34.379+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-26T07:20:34.380+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_weather, task_id=extract_data, run_id=scheduled__2024-11-25T00:00:00+00:00, execution_date=20241125T000000, start_date=20241126T072033, end_date=20241126T072034
[2024-11-26T07:20:34.403+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-26T07:20:34.422+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-26T07:20:34.424+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
